regtests/ww3_ufs1.1/input_unstr/ww3_outp.inp

```diff
@@ -1,59 +1,44 @@
 $ -------------------------------------------------------------------- $
-$ WAVEWATCH III NETCDF Point output post-processing                    $
+$ WAVEWATCH III Point output post-processing                           $
 $--------------------------------------------------------------------- $
 $ First output time (yyyymmdd hhmmss), increment of output (s), 
-$ and number of output times.
+$ and number of output times, optional per-time-step (1) or single (0)
+$ point output, optional point output prefix
 $
-  20210401 000000  3600.  100
+  20210401 000000  3600.  25  1 
```

Comment:
Please check the permissions changing on this file.  Also, please add a prefix here so that we can test that feature.

Reply:
Can we add a second ww3_outp to do both bull/cbull and spec files to test both features?